web-ng/src/app/components/drawing-tools/drawing-tools.component.scss

```diff
@@ -31,13 +31,13 @@
 }
 
 .add-point-button {
-  background-color: var(--drawing-tools-black);
+  background-color: mat-color($black);
```

Comment:
The custom colors were fine, I was just asking that you use either colors in the theme, or define a variable for colors if they're not present and needed. Could you please revert these changes?

Reply:
reverted